pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/db/dao/JdbiCompositeQuestion.java

```diff
@@ -31,4 +32,12 @@ default void insertChildren(long compositeQuestionId, List<Long> childQuestionId
     void insertChild(@Bind("parentQuestionId") long parentQuestionId, @Bind("childQuestionId") List<Long> childQuestionIds,
                      @Bind("orderIndex") List<Integer> orderIdxs);
 
+    @SqlQuery("SELECT child_question_id FROM composite_question__question where parent_question_id=:parentQuestionId")
+    List<Long> getChildQuestionIds(@Bind("parentQuestionId") long parentQuestionId);
+
+    @SqlUpdate("DELETE FROM composite_question__question where parent_question_id=:parentQuestionId")
+    int deleteChildQuestionMembership(@Bind("parentQuestionId") long parentQuestionId);
+
+    @SqlUpdate("DELETE FROM composite_question where question_id=:questionId")
```

Comment:
Adds whitespace to improve readability and consistent capitalization of keywords

```suggestion
    @SqlUpdate("DELETE FROM composite_question WHERE question_id = :questionId")
```